src/sentry/contexts.cpp

```diff
@@ -61,12 +62,14 @@ Dictionary make_device_context(const Ref<RuntimeConfig> &p_runtime_config) {
 		device_context["orientation"] = orientation;
 	}
 
-	// TODO: Need platform-specific solutions - this doesn't work well.
-	String host = OS::get_singleton()->get_environment("HOST");
-	if (host.is_empty()) {
-		host = "localhost";
+	if (SentryOptions::get_singleton()->is_send_default_pii_enabled()) {
+		// TODO: Need platform-specific solutions - this doesn't work well.
+		String host = OS::get_singleton()->get_environment("HOST");
+		if (host.is_empty()) {
+			host = "localhost";
+		}
+		device_context["name"] = host;
```

Comment:
Avoid sending data if it won't be useful, `localhost` here wont' add much

```suggestion
		if (!host.is_empty()) {
			device_context["name"] = host;
		}
```

Reply:
Gotcha, committed.